wasm/jsapi/wasm-module-builder.js

```diff
@@ -329,6 +369,108 @@ let kExprI32ReinterpretF32 = 0xbc;
 let kExprI64ReinterpretF64 = 0xbd;
 let kExprF32ReinterpretI32 = 0xbe;
 let kExprF64ReinterpretI64 = 0xbf;
+let kExprI32SExtendI8 = 0xc0;
+let kExprI32SExtendI16 = 0xc1;
+let kExprI64SExtendI8 = 0xc2;
+let kExprI64SExtendI16 = 0xc3;
+let kExprI64SExtendI32 = 0xc4;
+let kExprRefNull = 0xd0;
+let kExprRefIsNull = 0xd1;
+let kExprRefFunc = 0xd2;
+
+// Prefix opcodes
+let kNumericPrefix = 0xfc;
+let kSimdPrefix = 0xfd;
+let kAtomicPrefix = 0xfe;
```

Comment:
Is it fine to add new proposal's opcodes?

Reply:
The [phase document](https://github.com/WebAssembly/meetings/blob/master/process/phases.md) expects that tests are written during phase 2, so I think anything needed to support tests for phase &geq; 2 proposals is acceptable to add.